tck/docs/userguide/src/main/asciidoc/book.adoc

```diff
@@ -1,4 +1,4 @@
-= TCK User's Guide for Technology Implementors
+= TCK User's Guide for Technology Implementers
```

Comment:
Implementor is a valid alternate form of Implementer.

https://www.merriam-webster.com/dictionary/implementor

Reply:
I'll revert it